tox.ini

```diff
@@ -14,20 +14,25 @@ deps =
     pytest ~= 4.3.0
     pytest-cov ~= 2.6.1
     pytest-randomly ~= 1.2.3
-    python-kadmin ~= 0.1.2
+    git+https://github.com/twAlexM/python-kadmin.git@7b89b1880ad09b753c8366dad114b79674d00597
```

Comment:
maybe we should just fork this project?

Reply:
Ya... This definitely feels like a hack. I'm not too worried about it since it's test code, but if that fork ever goes away, it will break anything that ever depended on it (e.g. we won't be able to rerun old Travis builds). I think we can use the same commit from the original repo since there's a PR open for it (https://github.com/rjancewicz/python-kadmin/pull/59). At least if that goes away, we'd probably need to find a more permanent replacement anyways.

Reply:
Actually, that doesn't seem to work:
```
Collecting git+https://github.com/rjancewicz/python-kadmin.git@7b89b1880ad09b753c8366dad114b79674d00597
  Cloning https://github.com/rjancewicz/python-kadmin.git (to revision 7b89b1880ad09b753c8366dad114b79674d00597) to /tmp/pip-req-build-lsuj1890
  Running command git clone -q https://github.com/rjancewicz/python-kadmin.git /tmp/pip-req-build-lsuj1890
  Running command git checkout -q 7b89b1880ad09b753c8366dad114b79674d00597
  fatal: reference is not a tree: 7b89b1880ad09b753c8366dad114b79674d00597
ERROR: Command errored out with exit status 128: git checkout -q 7b89b1880ad09b753c8366dad114b79674d00597 Check the logs for full command output.
```

Reply:
> we won't be able to rerun old Travis builds

As it is, `python-kadmin` is not used in Travis because a KDC would be required.
In fact, it's only used in 1 place (for cleanup):
https://github.com/Isilon/isilon_hadoop_tools/blob/29d9936d90b859b40dd111e3fca550404f924378/tests/conftest.py#L354-L355

I'm gonna leave it for now, but I've asked the publisher of `kadmin` if they will publish the commit we're using. Let's keep an eye on it.